Trakt/Configuration/PluginConfiguration.cs

```diff
@@ -26,18 +29,20 @@ public PluginConfiguration()
     public TraktUser[] TraktUsers { get; set; }
```

Comment:
I can't remove the public array because otherwise the frontend has no access to this...

Reply:
But you could just use `_traktUsers` as the backing field and remove the setter?

Reply:
I can't because otherwise the plugin settings are not persisted on disk. Only public fields are saved.

Reply:
I removed the locking because saving the plugin settings via the plugin config server endpoint makes the locking useless.